### DISCUSSION TITLE
Unable to screenshot or screen record

### DISCUSSION THREAD

**Discussion by talormanda** [keepassxreboot/keepassxc#7652] [Q&A] [answered]
Hello. For demonstration purposes I want to be able to send a screenshot of the application and record video, but the application gets hidden for security purposes. Is there a way to temporarily disable this?

**Comment by droidmonkey** [comment, marked as answer]
https://keepassxc.org/docs/KeePassXC_UserGuide.html#_screenshot_security

**Reply by talormanda** [reply]
Thanks. I totally didn't find that when I was googling.

**Comment by jeffshead** [comment]
So on Windows, how do you set KeePassXC to auto-start, at system boot, with the `--allow-screencapture` switch?

**Reply by droidmonkey** [reply]
Add that switch to the startup entry in the registry. Unfortunately whenever you open and press ok on the application settings it will reset the registry entry.

**Reply by jeffshead** [reply]
I tried the following but it did not work:
![image](https://user-images.githubusercontent.com/349651/220164094-30370b77-c7a0-4fe8-adda-a53fde785b5b.png)
Maybe I did not edit the correct entry or specified the switch incorrectly???

I really need to have a solution that allows viewing KeePassXC remotely because it's for someone that routinely needs remote assistance.

**Reply by droidmonkey** [reply]
Put the flag outside the quotes.

**Reply by droidmonkey** [reply]
2.7.5 will include this feature: https://github.com/keepassxreboot/keepassxc/pull/8841

**Reply by jeffshead** [reply]
Thanks!
So as a workaround, I logged into each users Windows account, opened KeePassXC and unticked the option to start at login. Then I created a shortcut, for **KeePassXC.exe**,  with the `--allow-screencapture` swith and placed that shortcut into the following folder:
_C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Startup_
